### ISSUE TITLE
ZNM Key item disappeared

### ISSUE THREAD

**Issue by Sethhp** [HomepointXI/Issues#257]
**Date & Time**: 
oct 15.2018

**Client Version (use `/ver` in game):**
30180904_0

**Character Name:**
Seth

**Discord Name (if different from in game):**
Seth

**Nation:**
sandy

**Job(level)/Sub Job(level):**
99BLU/49RDM

**NPC or Monster or item Name:**
Sanraku

**Zone name:** 
Aht Urhgan Whitegate

**Coordinates (use `!where`):**

**Item tool link (for items issues only):**

**Multi-boxing? (multiple clients on same connection):**
yea

**Steps to reproduce:**
Was gathering pop item for Tinnin , killed both t3 for the item and got Tinnin pop item to show up in the column where u buy the pop item from, Sanraku. Then I went on to start doing the troll path after I came back and turn in the troll path item, Tinnin pop item is gone, before I started troll path I had all the KI =Charcoal Grey Seal , Deep Purple Seal, Chestnut-colored Seal in temp KI section, now it's only showing that I got Charcoal Grey Seal, Chestnut-colored Seal, on all my 7 Toons.  Deep Purple Seal is gone from the temp ki section, as well. I think some of the t3 pop item just disappear upon zoning out or something cause im missing a few in the troll path also.

**Comment by EpicTaru** [comment]
Was it an actual item in your inventory that disappeared or a KI that isn't showing anymore?

**Comment by Sethhp** [comment]
I never pulled the actual item out into inventory it was still in the section when I left for troll path, came back and the KI wasn't there anymore

**Comment by EpicTaru** [comment]
KK, thanks for clarifying.

**Comment by Sethhp** [comment]
yw.

**Comment by lonleyhero** [comment]
Possibly related I was farming bukhis last night and and at one point two of my characters that showed having all 3 pop items couldent pop and were missing the gargoyle hand after checking the ???

**Comment by munkee128** [comment]
Same issue with ZNM KI & Item disappearing.

Awhile back I did 6x Khromasoul Bhurborlor (farming for Burtgang). Traded that in and got the Keyitem (Gold-Colored Seal)
Monday, Jan 7th., did another 6x of the same ZNM (helping others farm for Burtgang) and this time held onto Bhurborlor's Vambrace (since I already had the KI from last time, or so I thought).

Come today, I'm trying to consolidate what KIs and what Items I have for ZNMs and I noticed that both the Gold-Colored Seal Keyitem and the Bhurborlor's Vambrace all disappeared (from all 6 characters)
